os/include/tinyara/binary_manager.h

```diff
@@ -70,6 +70,14 @@
 /* Mount point for User binaries */
 #define BINARY_DIR_PATH                  CONFIG_MOUNT_POINT"bins"
 
+/* The length of binary file or kernel partition path.
+ * A path is same like below:
+ *  - User filepath : "/<mount_path>/<binary_name>_<binary_version>"
+ *         length : CONFIG_NAME_MAX + BIN_NAME_MAX + binary_version (32) + /,_(3) + '\0'(1)
+ *  - Kernel partpath : "/dev/mtdblock#" defined as BINMGR_DEVNAME_FMT
+ */
+#define BINARY_PATH_LEN                  (CONFIG_NAME_MAX + BIN_NAME_MAX + 36)
```

Comment:
I think you'd better define 36 with strlen or some macro.

Reply:
It can't define value with function because it is used in array like line 221.